### PR TITLE
Update realm package metadata.

### PR DESCRIPTION
**Pull request by elle-j**
## What, How & Why?

Update metadata as a pre-step to creating a PR for updating Realm in [React Native Directory](https://reactnative.directory/).

Adding e.g. `storage` as a keyword in order to be listed in the React Native Directory when searching for "storage" as [Expo recommends](https://reactnative.directory/).

## ☑️ ToDos
--

### CODE REVIEW

File: packages/realm/package.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
 {
   "name": "realm",
   "version": "12.2.1",
-  "description": "Realm is a mobile database: an alternative to SQLite and key-value stores",
+  "description": "Realm by MongoDB is an offline-first mobile database: an alternative to SQLite and key-value stores",
```
Comment (kraenhansen) [inline packages/realm/package.json:4]: local first? @ianpward what terms are we using these days?

**Review by kraenhansen — APPROVED** [review]

File: packages/realm/package.json (hunk 1 above)
Comment (elle-j) [reply packages/realm/package.json:4]: Spoke to Ian, he said that "offline-first" is what we've traditionally gone with.

Comment (takameyer) [reply packages/realm/package.json:4]: Maybe we can add "local first" to the keywords?

**Review by takameyer — APPROVED** [review]
LGTM

File: packages/realm/package.json [hunk 2]
```diff
@@ -15,7 +16,8 @@
     "asyncstorage",
```
[the remark this answers is not in the document]
Comment (elle-j) [reply packages/realm/package.json:16]: Was wondering about this as well first. But searching for e.g. ["asyncstorage"](https://reactnative.directory/?search=asyncstorage) yields more results than ["async storage"](https://reactnative.directory/?search=async+storage). (See the [package.json](https://github.com/artxty/typed-async-storage/blob/08e9577d0b8a21a741ca5764ab47b669b73ab41d/package.json#L8) of the top result)

I mean we could add both variants, unless the total number of keywords has a negative affect in some way.

[the remark this answers is not in the document]
Comment (takameyer) [reply packages/realm/package.json:16]: Both variants sounds good.  Just trying to think about what one would type in a search bar.  There doesn't seem to be any limit or negative impact, outside of just adding more lines of code to the `package.json`

[the remark this answers is not in the document]
Comment (elle-j) [reply packages/realm/package.json:18]: ✅

**Pull request merged**